eo-runtime/src/main/java/org/eolang/Main.java

```diff
@@ -187,11 +187,10 @@ private static void run(final List<String> opts) throws Exception {
             phi.attr("Δ").put(new Data.Value<>(arg));
             app.attr(0).put(phi);
         }
-        if (!new Dataized(app).take(Boolean.class)) {
-            throw new IllegalStateException(
-                "Runtime dataization failure"
-            );
-        }
+        final Object result = new Dataized(app).take();
+        Main.LOGGER.info(
+            String.format("The result of %s dataization is %s", app, result.getClass())
```

Comment:
@yasamprom but now the data is not printed :) First, think about the user, the functionality of the product, the meaning of what you are doing. Then, think about getting your PR being accepted.

Reply:
Done